devel/fortran-utils/Portfile

```
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+
+name            fortran-utils
+version         1.2
+revision        0
+categories      devel
+platforms       darwin
+license         BSD
+maintainers     {@kamischi web.de:karl-michael.schindler} openmaintainer
+homepage        https://sourceforge.net/projects/fortran-utils/
+master_sites    sourceforge
+use_bzip2       yes
+extract.mkdir   yes
+checksums       rmd160  353e753b2084f3eab5cf9cfd9c2ccf4e983f05b5 \
+                sha256  b2a25ae867a7755fb8d839ca5dd59a24888d12d217aea6ddfd57aceda888b197 \
+                size    7689
+
+description     Tools for use with Fortran77 code
+
+long_description \
+    The fortran-utils are: \
+    \n- fpr: transform Fortran's carriage control conventions to UNIX line printer conventions \
+    \n- fsplit: split a multi-routine Fortran file into individual files
+
+notes "This version has patches from FreeBSD and NetBSD."
+
+installs_libs   no
+use_configure   no
+build {
+    foreach p [list fpr fsplit] {
+        system -W ${worksrcpath}/${p} "${configure.cc} ${configure.cflags} ${configure.ldflags} [get_canonical_archflags cc] -o ${p} ${p}.c"
+    }
+}
+
+destroot {
+    xinstall -m 0755 -W ${worksrcpath} fpr/fpr fsplit/fsplit ${destroot}${prefix}/bin
+    xinstall -m 0444 -W ${worksrcpath} fpr/fpr.1 fsplit/fsplit.1 ${destroot}${prefix}/share/man/man1/fsplit.1
```

Oops, I didn't quite proof-read that before making my suggestion. It actually needs to be:
```suggestion
    xinstall -m 0444 -W ${worksrcpath} fpr/fpr.1 fsplit/fsplit.1 ${destroot}${prefix}/share/man/man1/
```
although this will be superseded by my version using the Makefiles.

---

If you did this already, go ahead. I am not fixed to my version.